.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Thanks for doing the requested changes but please remove the settings.json file. Let's stick to the main purpose of the project and the PR. Enforcing specific theme colors for VSCode is not ok for me and it should be up to the users or the theme they have selected.

Reply:
Sorry, this got committed without me thinking about it. I've dropped it and updated the README.